services/course-runner-pod.yaml

```diff
@@ -0,0 +1,23 @@
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: kp-course-runner-
+spec:
+  containers:
+    - name: runner-container
+      image: "docker-registry.default.svc:5000/kielipankki-github-runners/course-runner:v0.0.2"
+      workingDir: /home/kprunner
+      args: ["$(GITHUB_PAT)"]
+      env:
+        - name: GITHUB_PAT
+          valueFrom:
+            secretKeyRef:
+              name: runner-secrets
+              key: gh_runner_pat
+              optional: false
+        - name: RUNNER_REPO
+          value: "aajarven/Kielipankki"
```

Comment:
Should this be parametrised?

Reply:
Maybe not here (not likely that we'll have another course material repo in the foreseeable future, I suppose), as it's pretty much that we change it to csc-training/Kielipankki once and then never touch it. But in the long term (when we have more repositories with automated tests) the python-runner might benefit from getting templated.